Image img
Float32MultiArray depth
---
Image segmap